Kernel/Heap/Heap.h

```diff
@@ -20,33 +20,25 @@ class Heap {
 
     struct AllocationHeader {
         size_t allocation_size_in_chunks;
-#if ARCH(X86_64)
-        // FIXME: Get rid of this somehow
-        size_t alignment_dummy;
-#endif
-        u8 data[0];
+        alignas(void*) u8 data;
```

Comment:
please add the `[0]` or just `[]` back in, it isnt just a u8, but essentially a vla